tests/exchange/test_bitget.py

```diff
@@ -360,20 +353,20 @@ def test_bitget_ohlcv_candle_limit(default_conf, mocker):
     start_time = int(datetime(2021, 1, 1, tzinfo=timezone.utc).timestamp() * 1000)
 
     for timeframe in timeframes:
-        assert exchange.ohlcv_candle_limit(timeframe, CandleType.SPOT) == 1000
-        assert exchange.ohlcv_candle_limit(timeframe, CandleType.FUTURES) == 1000
+        assert exchange.ohlcv_candle_limit(timeframe, CandleType.SPOT) == 500
```

Comment:
This is incorrect - bitget yields 1000 candles.
If your (local!) environment yields 500 candles, you'll have to update ccxt to match the verison of this branch - as your ccxt version then doesn't have the "features" object yet - which suggests it's a pretty old version, too.